diagnostics/forcing_feedback/forcing_feedback.html

```diff
@@ -0,0 +1,59 @@
+<!-- This file is part of the html module of the MDTF code package (see mdtf/MDTF_v2.0/LICENSE.txt) -->
+
+<IMG SRC="../mdtf_diag_banner.png" border=0 usemap="#logo">
+<TITLE>Forcing and Feedbacks</TITLE>
+<!-- <H3>Convective Transition Statistics</h3> -->
+<P>
+The Forcing and Feedback module uses radiative kernels to compute the model's Instantaneous Radiative Forcing and the Planck, lapse rate, water vapor, surface albedo and cloud radiative feedbacks. The total, surface-temperature-mediated radiative change (also known as the feedback parameter) is also computed. These quantities are defined at the top-of-the-atmosphere (TOA) and decomposed into longwave (LW) and shortwave (SW) components when applicable. Radiative feedbacks (in W/m2/K) are diagnosed by regressing local radiative anomalies by global-mean surface temperature. 
+
+These results can be used to understand what is driving the a model's particular climate sensitivity and more broadly provide insights on how changes in the model's atmospheric state alter the model's energy budget.
+<p>
+<!--<a href="MDTF_Documentation_convective_transition.pdf">Full Documentation and Contact Information</a> -->
+<P>
+<TABLE>
+<TR>
+<TH ALIGN=LEFT><font color=navy> Forcing and Feedbacks
+<TH ALIGN=LEFT>{CASENAME}
+<TH ALIGN=LEFT>OBS
+<TR>
+<TH ALIGN=LEFT>Global-Mean Total Radiation Change
+<TH ALIGN=CENTER><A HREF=model/forcing_feedback_globemean_Rad.png>plot</A>
```

Comment:
- Table formatted for three columns, but entries for obs data plots are missing
- `<td>` is the HTML tag for table entries; `<th>` is used for row/column headers (probably still formats correctly)
- Table rows should be closed with `<\tr>`

Reply:
Thanks for catching this. I'm new to HTML and am simply looking to display a bunch of plots with a title above each of them. I just followed the formatting of lines 28-41 from the example POD html (just doing so with more plots). Is this the wrong formatting to follow? Is there an HTML file from another POD I should look at instead?